install

```diff
@@ -118,11 +118,17 @@ do_install()
     sudo rosdep init
   fi
   if [ ! -f /etc/ros/rosdep/sources.list.d/10-ros-install-osx.list ]; then
-    echo "This sudo prompt adds the the brewed python rosdep yaml to /etc/ros/rosdep/sources.list.d/10-ros-install-osx.list"
+    echo "This sudo prompt adds the brewed python rosdep yaml to /etc/ros/rosdep/sources.list.d/10-ros-install-osx.list"
     sudo sh -c "echo 'yaml file://$(pwd)/rosdeps.yaml osx' > /etc/ros/rosdep/sources.list.d/10-ros-install-osx.list"
   fi
   rosdep update
 
+  # set up the symlink for openssl
+  # based on https://stackoverflow.com/questions/45693149/openssl-aes-h-file-not-found-on-mac
+  cd /usr/local/include
+  ln -fs ../opt/openssl/include/openssl .
+  cd -
```

Comment:
I had fixed this with adding some `find_package(OpenSSL)` in the cmake lists. But it is not clean yet (doesn't propagate to depending packages), so I'm not sure if it is worth the effort if this also works just as well.

Reply:
PS: I noticed another issue with this. Since you only set the include path, the linker still links to the system ssl and crypto libraries. I guess that might lead to wired issues...

```
$ otool -L /opt/ros/melodic/lib/librosbag_storage.dylib | grep /usr/lib
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1238.60.2)
	/usr/lib/libbz2.1.0.dylib (compatibility version 1.0.0, current version 1.0.5)
	/usr/lib/libssl.0.9.8.dylib (compatibility version 0.9.8, current version 0.9.8)
	/usr/lib/libcrypto.0.9.8.dylib (compatibility version 0.9.8, current version 0.9.8)
	/usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 307.5.0)
```

Reply:
I got around that by passing `-DCMAKE_CXX_FLAGS="-I/usr/local/opt/openssl/include"` to it.

Reply:
@Karsten1987, you mean as an alternative to `ln -fs ...` or adding `find_package ...`? I guess it doesn't address the linker issue still?

Reply:
As an alternative to the `ln -fs`. For finding the library in the first place, I added the openssl to my `DYLD_LIBRARY_PATH`:

```
export DYLD_LIBRARY_PATH=`brew --prefix openssl`/lib:$DYLD_LIBRARY_PATH
```

Reply:
Ah yes that would fix it also for dynamic linking. Thanks! 👍